experiments/compositions/scripts/github-actions/getter-test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export PATH=$PATH:$(go env GOPATH)/bin
+cd experiments/compositions/composition
+GCP_PROJECT_ID=something make unit-test-expander-getter
```

Comment:
is this temporary :)

Reply:
No this is permanent for the kind based testing in github. Locally we have a GCP project context. In the github runner we dont have one. So setting it to a dummy value so that we dont have to change our scripts.